Python/other/Subsequence_check.py

```diff
@@ -0,0 +1,38 @@
+# Take input for class TestCase
+print("Enter Number of TestCase :")
+testcase= int(input())
```

Comment:
@yashaswibiyahut Update this to `testcase = int(input())`

Reply:
done @iamrajiv 
kindly review it.